src/train.py

```diff
@@ -138,6 +138,7 @@ def main(args):
 
     # Logging
     logger = Logger(_logger)
+    args = flatten(args)
```

Comment:
Wouldn't override `args` since we need it in other parts in the code. Would rename and create new variable